com.unity.netcode.gameobjects/Runtime/Messaging/RpcParams.cs

```diff
@@ -96,7 +96,7 @@ public struct ClientRpcParams
 
 #pragma warning disable IDE1006 // disable naming rule violation check
     // RuntimeAccessModifiersILPP will make this `public`
-    internal struct __RpcParams
+    public struct __RpcParams
```

Comment:
@ShadauxCat 
This update will force us to increment the minor version.
Was this the final solution for this particular ILPP issue?

Reply:
Oh, I didn't actually notice that change when I was looking at it. Or... I did it temporarily and forgot to address it. One of the two... I'll take another look...